.github/workflows/ci.yml

```diff
@@ -9,6 +9,75 @@ on:
     #branches: [ "master" ]
 
 jobs:
+  aswf_old:
```

Comment:
Is that like a backup of previous aswf: which you've modified? How would this aswf_old be used?